### PR TITLE
JSONize mending modifier from mutations

### PR DESCRIPTION
**Pull request by anothersimulacrum**
#### Summary
SUMMARY: None

#### Purpose of change
The effect mutations have on limb mending was hardcoded in a if/else ladder. Yuck! Also, some of the slow healer mutations were not having an impact on mending speed, so add that.

#### Describe the solution
Add `mending_modifer` value to mutations, of type float.
Add code to load this, and replace the if/else ladder with multiplying times this multiplier.
Add this value to mutations.

#### Testing
Break an arm, and debug in the reptilian healing mutation. Sleep, and notice that your arm mends very quickly, as it should.

### CODE REVIEW

File: doc/JSON_INFO.md [hunk 1]
```diff
@@ -1130,6 +1130,7 @@ Note that even though most statistics yield an integer, you should still use
 "fatigue_regen_modifier": 0.333, // Modifier for the rate at which fatigue and sleep deprivation drops when resting.
 "healing_awake": 1.0, // Healing rate per turn while awake.
 "healing_resting": 0.5, // Healing rate per turn while resting.
+"mending_modifier": 1.2 // Multiplier on how fast you heal - This value would make you heal 20% faster
```
Comment (Night-Pryanik) [inline doc/JSON_INFO.md]: I'd rephrase it as "how fast your limbs mend".

Comment (Ramza13) [reply doc/JSON_INFO.md]: But your head and torso aren't limbs?

Comment (Zireael07) [reply doc/JSON_INFO.md]: 'How fast your body mends'?
